Makefile

```diff
@@ -34,25 +34,20 @@ OUT_DIR=internal/api
 proto:
 	protoc --proto_path=$(PROTO_DIR) --go_out=$(OUT_DIR) --go-grpc_out=$(OUT_DIR) --golang-deepcopy_out=:$(OUT_DIR) $(PROTO_DIR)/**/*.proto
 
-.PHONY: gen-istio-manifests
-gen-istio-manifests:
-	bash test/scripts/generate_istio_manifests.sh $(ISTIO_VERSION)
-
 .PHONY: kind-clusters
 kind-clusters:
 	bash test/scripts/kind_provisioner.sh $(ISTIO_VERSION)
 
-.PHONY: e2e-test
-e2e-test:
-	go test -tags=integ -run TestTraffic ./test/e2e \
-		--istio.test.hub=docker.io/istio\
-		--istio.test.tag=$(ISTIO_VERSION)\
-		--istio.test.kube.config=$(shell pwd)/test/east.kubeconfig,$(shell pwd)/test/west.kubeconfig\
-		--istio.test.kube.networkTopology=0:east-network,1:west-network\
-		--istio.test.onlyWorkloads=standard
-
 .PHONY: e2e
-e2e: kind-clusters e2e-test
+TEST_SUITES ?= mcp k8s
+e2e: kind-clusters # kind-clusters target should not fail when clusters already exist
+	@$(foreach suite, $(TEST_SUITES), \
+		go test -tags=integ -run TestTraffic ./test/e2e/$(suite) \
+			--istio.test.hub=docker.io/istio\
+			--istio.test.tag=$(ISTIO_VERSION)\
+			--istio.test.kube.config=$(shell pwd)/test/east.kubeconfig,$(shell pwd)/test/west.kubeconfig\
+			--istio.test.kube.networkTopology=0:east-network,1:west-network\
+			--istio.test.onlyWorkloads=standard;)
```

Comment:
Do you want to implement a KinD cluster tear-down at the end of `e2e`, with an option for the user to skip it? Or maybe the opposite would be more useful: Tear down the clusters only if the user has set some variable like `TEAR_DOWN_CLUSTERS=true`?

Reply:
I will consider this option in the next PR.